src/tools/openpgp-tool.c

```diff
@@ -479,14 +588,16 @@ int main(int argc, char **argv)
 	r = util_connect_card(ctx, &card, opt_reader, opt_wait, verbose);
 	if (r) {
 		util_fatal("failed to connect to card: %s\n",
-			sc_strerror(r));
+		           sc_strerror(r));
```

Comment:
Either change the convention for the whole file or use the current convention (tabs only). Aligning that matches the opening bracket isn't used anywhere else, as far as I know.